src/Illuminate/Collections/Arr.php

```diff
@@ -393,6 +393,21 @@ public static function isAssoc(array $array)
         return array_keys($keys) !== $keys;
     }
 
+    /**
+     * Determines if an array is multidimensional.
+     *
+     * @param  array  $array
+     * @return bool
+     */
+    public static function isMultidimensional(array $array)
+    {
+        foreach ($array as $value) {
+            return is_array($value);
+        }
```

Comment:
No need to use a for loop when only the first item in the array is being checked. Then, what if the first item of my array is a string and the second is another array. I would consider it multidimensional but the function would return false.

Reply:
Yes that is true, fixed and test extended with your example.